src/components/draw/feature-sub-content.vue

```diff
@@ -20,35 +20,51 @@ const emit = defineEmits([
   'toggleEditFeature',
   'toggleDock',
   'clickDelete',
-  'submitEditInfo',
+  'resetInfo',
+  'resetStyle',
+  'submitEditFeature',
 ])
 const { t } = useTranslation()
-const feature: DrawnFeature | undefined = inject('feature')
+const feature: DrawnFeature = inject('feature')!
+let prevLabel = feature.label
+let prevDescription = feature.description
+// keep deep copy of previous style to be able to revert style on cancel
+const prevStyle: DrawnFeatureStyle = { ...feature.featureStyle }
```

Comment:
What is the logic in keeping the style or the description + label in this specific component? You handle the change in the parent component, it is very hard to find where an operations an setters are made.

Reply:
in case you cancel the editions, label and description are restored. This is not related to the PR, but it was a functional difference with prod

Reply:
Yes I understand the functionality, my question was storing previous values here in this specific feature-sub-content.vue? what is the link with "sub-content"? You could just emit the event validate and cancel, and the parent handles the previous values itself.

Plus I see the label is updated in the treeview but not on the map.